sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 ## 5.1.0-beta.2 (Unreleased)
-
+### Breaking changes
+- Removed property `length` from `CategorizedEntity`, `SentenceSentiment`, `LinkedEntityMatch`, `AspectSentiment`, 
```

Comment:
Could you also add why we are removing this in the PR description? Its helps in future referencing this.

Reply:
Updated

Reply:
Thanks !